src/app-layout/visual-refresh-toolbar/drawer/global-drawer.tsx

```diff
@@ -75,8 +75,9 @@ function AppLayoutGlobalDrawerImplementation({
     (wasExpanded && !isExpanded);
 
   return (
-    <Transition nodeRef={drawerRef} in={show || isExpanded} appear={show || isExpanded} timeout={0}>
-      {state => {
+    <Transition in={show} exit={!show}>
```

Comment:
Note: Removed `isExpanded` condition because when 'isExpanded' it's always `show`.

Reply:
Note: The appear props is not available in the internal Transition component. If necessary we can introduce it, but before that I want to discuss if it's necessary.